### PR TITLE
Use the System hub category provided by Anaconda

### PR DESCRIPTION
**Pull request by M4rtinK**
Anaconda now provides the Security category, including correctly set
sortOrder. So use it instead and remove the local copy.

Related: rhbz#1269211

### CODE REVIEW

**Comment by M4rtinK** [comment]
The related Anaconda PR: https://github.com/rhinstaller/anaconda/pull/632

**Comment by bcl** [comment]
That's certainly an improvement. I also think it would look just fine as part of the SYSTEM section. I don't really think we need a whole new section for just 1 addon, especially since the title of it is 'Security Policy'

**Comment by vpodzime** [comment]
Looks good to me.

**Comment by vpodzime** [comment]
> That's certainly an improvement. I also think it would look just fine as part of the SYSTEM section. I don't really think we need a whole new section for just 1 addon, especially since the title of it is 'Security Policy'

I would have no problem with this either. There seems to be no other initiatives that could lead to more security-related addons.

**Comment by M4rtinK** [comment]
I've updated the pull request to use the System category. :)

**Comment by vpodzime** [comment]
> I've updated the pull request to use the System category. :)

Doesn't really look like you did. Something must have gone wrong.

**Comment by M4rtinK** [comment]
>Doesn't really look like you did. Something must have gone wrong.
Indeed! Let's try again. :)

**Comment by M4rtinK** [comment]
Looks correct to me now.

File: org_fedora_oscap/gui/spokes/oscap.py [hunk 1]
```diff
@@ -41,6 +40,7 @@
 from pyanaconda.ui.communication import hubQ
 from pyanaconda.ui.gui.utils import gtk_action_wait, really_hide, really_show
 from pyanaconda.ui.gui.utils import set_treeview_selection, fire_gtk_action, GtkActionList
+pyanaconda.ui.categories.system import SystemCategory
```
Comment (vpodzime) [inline org_fedora_oscap/gui/spokes/oscap.py]: ``from`` missing here?

**Comment by vpodzime** [comment]
> Looks correct to me now.

Not really. ;)

**Comment by M4rtinK** [comment]
> Not really. ;)
Bah! Third time is the charm I guess. :)

**Comment by vpodzime** [comment]
Looks good to me.

**Pull request merged**